test/performance_tests/performance_test_image_function.cpp

```diff
@@ -327,6 +335,7 @@ namespace image_function
     SET_FUNCTION( LookupTable        )
     SET_FUNCTION( Maximum            )
     SET_FUNCTION( Minimum            )
```

Comment:
Please a test for function_pool as well as it uses your new code internally. It should work even faster. Additionally, please align **)** symbol with other SET_FUNCTION lines by white space shift :)